model_compression_toolkit/keras/keras_implementation.py

```diff
@@ -119,7 +151,16 @@ def shift_negative_correction(self,
         """
         return apply_shift_negative_correction(graph,
```

Comment:
I think it better to move this function to  _compression_toolkit/keras/graph_substitutions/substitutions/shift_negative_activation.py
In the following meaner: 
def kears_apply_shift_negative_correction( graph,qc):
       return apply_shift_negative_correction(graph, ...)